js/main.js

```diff
@@ -160,6 +160,11 @@ mobileMenuBtn.addEventListener("click", () => {
 	menuIconOpen.classList.toggle("hidden");
 	menuIconClose.classList.toggle("hidden");
 
+	if (!menuIconOpen.classList.contains("hidden")) {
```

Comment:
updating aria label after mobile menu icons and buttons are toggled

Reply:
Nice!